packages/did-ipfs-store/src/didStore.ts

```diff
@@ -48,6 +68,42 @@ export class DidStore implements IDidStore {
     }
   }
 
+  /**
+   * Checks if file is pinned on cluster
+   * @param cid CID
+   * @param replication specifies on how many nodes data should be pinned https://ipfscluster.io/documentation/guides/pinning/#replication-factors
+   */
+  async isPinned(cid: string, replication = REPLICATION.MIN): Promise<boolean> {
+    const { replication_factor_min, replication_factor_max } =
+      await this.allocations(cid);
+    const nodes =
```

Comment:
a little note: at first sight it is unclear for me if this constant contains list of nodes or number of nodes. I needed to analyze the code to figure it out.

Reply:
That's right. I coundn't figured clear name for number of nodes expected to pin data on. How about `expectedReplication`?

Reply:
or: `nodesCountToBeReplicatedTo` if it is not too long

Reply:
How about nodeReplicationCount?

Reply:
the same dwawback - node and count are part of replication

Reply:
maybe adding explicit type to this constant will help understand this?
```typescript
const expectedReplication: number =
```

Reply:
for me its the same as adding `count` to `replication`

Reply:
in order to make the difference from the process more obvious, the name can be lengthened like `expectedReplicationFactor`